ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+### Problem:
+
+### Expected results:
+
+### Actual results:
+
+### Steps to reproduce the problem:
+
+### Link to the page you are looking at:
+
+### Screenshots and any relevant files (please attach or link to them):
```

Comment:
I would call that `Notes, screenshots and other relevant files (please attach or link to them)`

I feel like a separate section with notes of things that might or might not be relevant could be helpful. That's what QA is using too, e.g https://github.com/mozilla/addons-server/issues/3295